src/components/axis.js

```diff
@@ -1,4 +1,19 @@
-export default function axis() {
+export default function axis(layout, flags) {
+  const axisComponent = flags.isEnabled('CLIENT_IM_2022')
+    ? (layout.components || []).find((c) => c.key === 'axis').axis
+    : {};
+  const labelStyle =
+    axisComponent.label && axisComponent.label.name
+      ? {
+          settings: {
+            labels: {
+              fontSize: axisComponent.label.name.fontSize,
+              fontFamily: axisComponent.label.name.fontFamily,
+              fill: axisComponent.label.name.fontColor.color,
```

Comment:
`fontColor` is `undefined` and  throws an exception when I try to change "font size" for the Axis label